migrations/1620879168566-CreateRfqmQuoteTable.ts

```diff
@@ -0,0 +1,34 @@
+import { MigrationInterface, QueryRunner, Table, TableIndex } from 'typeorm';
+
+export class CreateRfqmQuoteTable1620879168566 implements MigrationInterface {
+    name = 'CreateRfqmQuoteTable1620879168566';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.createTable(
+            new Table({
+                name: 'rfqm_quotes',
+                columns: [
+                    { name: 'order_hash', type: 'varchar', isPrimary: true },
+                    { name: 'metatransaction_hash', type: 'varchar', isUnique: true, isNullable: true },
+                    { name: 'created_at', type: 'timestamptz', default: 'NOW()' },
+                    { name: 'chain_id', type: 'varchar' },
+                    { name: 'integrator_id', type: 'varchar', isNullable: true },
+                    { name: 'maker_uri', type: 'varchar', isNullable: true },
```

Comment:
this is essential to RFQM later in the process, could go not nullable

Reply:
good call, just added some logic to allow for saving the `maker_uri` and made it not nullable